File: python/run_game.py
#from candyland import game
from candyland.game import Game

count = 10 #dummy value to emulate do-while loop

while count > 4:
	print "How many players?"
	count = int(raw_input())

mygame = Game(1)
mygame.play_game()
#mycard = Card("Red","",1)

#cardstring = mycard.to_string()
#print cardstring

File: java/RunGame.java
import candyland.Game;

public class RunGame {
	public static void main(String[] args) {
		System.out.println("Hello");
		int players;
		do {
			System.out.println("Number of players (4 max)?");
			players = Integer.parseInt(System.console().readLine());
		} while (players > 4);

		Game mygame = new Game(players);
		mygame.play_game();

	}
}